Description: Change CIM Schema default location to fit cim-schema's package location.
Forwarded: not-needed
Author: Thierry Carrez <thierry.carrez@ubuntu.com>
Origin: Ubuntu
Last-Update: 2010-06-28

Index: sblim-sfcb-1.3.8/sfcbrepos.sh.in
===================================================================
--- sblim-sfcb-1.3.8.orig/sfcbrepos.sh.in
+++ sblim-sfcb-1.3.8/sfcbrepos.sh.in
@@ -59,7 +59,7 @@ then
     echo -e "\t-X create repository in non-native format as specifed by argument"
     echo -e "\t-s specify staging directory [@localstatedir@/lib/sfcb/stage]"
     echo -e "\t-r specify repository directory [@localstatedir@/lib/sfcb/registration]"
-    echo -e "\t-c specify directory containing CIM Schema MOFs [@datadir@/sfcb/CIM]"
+    echo -e "\t-c specify directory containing CIM Schema MOFs [@datadir@/cim-schema]"
     echo -e "\t-t create tiny class repository by omitting inheritance information"
     echo -e "\t-z compress repository with gzip"
     echo
@@ -99,7 +99,7 @@ fi
 
 if [ -z "$cimschemadir" ]
 then
-    cimschemadir=${DESTDIR}@datadir@/sfcb/CIM
+    cimschemadir=${DESTDIR}@datadir@/cim-schema
 fi
 
 if [ -d $stagingdir ] && [ -f $stagingdir/default.reg ] &&
